scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -118,6 +116,45 @@ public static BuildTargetResult PublishDebFileToDebianRepo(BuildTargetContext c)
             return c.Success();
         }
 
+        [Target]
+        [Environment("DOCKER_HUB_REPO")]
+        [Environment("DOCKER_HUB_TRIGGER_TOKEN")]
+        public static BuildTargetResult TriggerDockerHubBuilds(BuildTargetContext c)
+        {
+            string dockerHubRepo = Environment.GetEnvironmentVariable("DOCKER_HUB_REPO");
+            string dockerHubTriggerToken = Environment.GetEnvironmentVariable("DOCKER_HUB_TRIGGER_TOKEN");
+
+            Uri baseDockerHubUri = new Uri("https://registry.hub.docker.com/u/");
+            Uri dockerHubTriggerUri;
+            if (!Uri.TryCreate(baseDockerHubUri, $"{dockerHubRepo}/trigger/{dockerHubTriggerToken}/", out dockerHubTriggerUri))
+            {
+                return c.Failed("Invalid DOCKER_HUB_REPO and/or DOCKER_HUB_TRIGGER_TOKEN");
+            }
+
+            c.Info("Triggering automated DockerHub builds for " + dockerHubRepo);
+            using (HttpClient client = new HttpClient())
+            {
+                StringContent requestContent = new StringContent("{\"build\": true}", Encoding.UTF8, "application/json");
+                try
+                {
+                    HttpResponseMessage response = client.PostAsync(dockerHubTriggerUri, requestContent).Result;
+                    if (!response.IsSuccessStatusCode)
+                    {
+                        var sb = new StringBuilder();
```

Comment:
<nit> Be consistent in your coding style.  Why use `var` here but not when declaring `baseDockerHubUri`, `requestContent`, or `client`?